client/src/modules/user/AuthBox/components/SignUpForm.jsx

```diff
@@ -268,6 +311,26 @@ class SignUpForm extends PureComponent {
         <FormattedMessage id="app.footer.terms-of-use" />
       </Link>
     );
+    const policyLabel = (
+      <FormattedMessage
+        id="user.auth.privacy-policy-agreement"
+        values={{
+          cookieLink,
+          privacyLink,
+          termsLink
+        }}
+      />
+    );
+    const policyErrorMessage = policyError ? (
+      <FormattedMessage
+        id="user.auth.input.policy-agreement-required"
+        values={{
+          cookieLink,
+          privacyLink,
+          termsLink
+        }}
+      />
+    ) : null;
```

Comment:
why null?

do just like this:

`policyError && (Element to return )`

Reply:
because it is passed to the child component: `AuthCheckbox` as a prop, where:
```
 {errorMessage && (
      <p className="auth-checkbox__feedback">
        <span className="auth-checkbox__icon">
          <ErrorIcon />
        </span>
        {errorMessage}
      </p>
    )}
```

Without this, I should pass the message and additional boolean to decided if this message has to be displayed or not